script.js

```diff
@@ -0,0 +1,116 @@
+const inputJs = document.querySelector(".input"),
+    toDoNewJs = inputJs.querySelector("#todo-new"),
+    listWaitingJs = document.querySelector(".waiting"),
+    listCompleteJs = document.querySelector(".complete");
+
+const TODOS = "toDos"; //key for saving todos in local storage
+
+let toDos = [];
+let maxId = 0;
+let countWaiting = 0, countComplete = 0;
+let tmpType; //only for deleteToDo and moveToDo
+
+function saveToDos() {
+    localStorage.setItem(TODOS, JSON.stringify(toDos));
+}
+
+function printCount() {
+    const countWaitingJs = document.querySelector("#count-waiting"),
+        countCompleteJs = document.querySelector("#count-complete");
+    countWaitingJs.innerText = countWaiting;
+    countCompleteJs.innerText = countComplete;
+}
+
+function deleteToDo(event){
+    const li = event.target.parentNode;
+    tmpType = li.parentNode.className;
+    
+    if(tmpType === "list waiting"){ 
+        listWaitingJs.removeChild(li);
+        countWaiting -= 1; 
+    } 
+    else{
+        listCompleteJs.removeChild(li);
+        countComplete -= 1;    
+    }
+    printCount();
+   
+    const cleanToDos = toDos.filter(function(toDo){
+        return toDo.id !== parseInt(li.id);
```

Comment:
```suggestion
    const cleanToDos = toDos.filter( toDo => toDo.id !== parselnt(li.id); )
```
이런 화살표 함수는 어떠세요? 화살표 함수가 간결하고 보기 좋아서 권장되는 것 같아요 !!

Reply:
훨씬 낫네요!!!!!!!!!!!! 너무좋아요 감사합니다. ㅎ